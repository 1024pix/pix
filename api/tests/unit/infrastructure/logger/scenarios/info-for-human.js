const sinon = require('sinon');

// before
// eslint-disable-next-line no-restricted-syntax
const fakedDate = new Date(0);
const clock = sinon.useFakeTimers(fakedDate);

// given
const settings = require('../../../../../lib/config');
settings.logging.enabled = true;
settings.logging.logLevel = 'info';
settings.logging.logForHumans = true;
const logger = require('../../../../../lib/infrastructure/logger');

// when
logger.info('message');

// after
clock.restore();
