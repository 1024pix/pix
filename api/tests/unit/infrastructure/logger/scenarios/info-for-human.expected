[00:00:00] [32mINFO[39m: [36mmessage[39m
